src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -74,7 +74,7 @@ class TezosPlatformDiscoveryOperations(
   /** Pre-caching attributes without cardinality */
   private def preCacheAttributes: DBIO[AttributesCache] = {
     val result = for {
-      tables <- MTable.getTables(Some(""), Some("public"), Some(""), Some(Seq("TABLE")))
+      tables <- MTable.getTables(Some(""), Some(""), Some(""), Some(Seq("TABLE")))
```

Comment:
I would expect this to be "public" -> "tezos", how comes it's now empty?

Reply:
you are right, changed